webapp.html

```diff
@@ -0,0 +1,133 @@
+    <!-- Just a quick notice to the user that it can be interacted with -->
+    <span class="drag-notice" id="js-drag-notice">Drag to rotate 360&#176;</span>
+    <!-- These toggle the the different parts of the chair that can be edited, note data-option is the key that links to the name of the part in the 3D file -->
+    <div class="options">
+        <div class="option --is-active" data-option="top">
+            <img src="shirt.jpeg" alt="" />
+        </div>
+        <div class="option" data-option="jacket">
+            <img src="jacket.png" alt="" />
+        </div>
+        <div class="option" data-option="pants">
+            <img src="pants.png" alt="" />
+        </div>
+
+<!--     
+        <div class="option" data-option="pants">
+            <img src="pants_svg.png" alt="" />
+        </div>
+        <div class="option" data-option="pedj">
+            <img src="https://s3-us-west-2.amazonaws.com/s.cdpn.io/1376484/back.svg" alt="" />
+                    </div>
+
+ -->        
+ 
+    </div>
+
+    <!-- The canvas element is used to draw the 3D scene -->
+ <canvas id="c"></canvas> 
+
+
+  <!--  <model-viewer id="shirt" camera-controls touch-action="pan-y" src="chair.glb" ar alt="A 3D model of a helmet"> -->
+
+
+    <div class="controls">
+        <div class="info">
+            <div class="info__message">
+                <p><strong>&nbsp;Grab&nbsp;</strong> to rotate chair. <strong>&nbsp;Scroll&nbsp;</strong> to zoom. <strong>&nbsp;Drag&nbsp;</strong> swatches to view more.</p>
+            </div>
+        </div>
+        <!-- This tray will be filled with colors via JS, and the ability to slide this panel will be added in with a lightweight slider script (no dependency used for this) -->
+        <div id="js-tray" class="tray">
+            <div id="js-tray-slide" class="tray__slide"></div>
+        </div>
+    </div>
+    <div class="frame">
+		<h1 class="frame__title">u-Wear: 3D Customizer App</h1>
+	</div>
+    <!-- The main three.js file -->
+    <script src='https://unpkg.com/three@0.127.0/build/three.js'></script>
+    <!-- This brings in the ability to load custom 3D objects in the .gltf file format. Blender allows the ability to export to this format out the box -->
+    <script src='https://cdn.jsdelivr.net/gh/mrdoob/three.js@r92/examples/js/loaders/GLTFLoader.js'></script>
+    <!-- This is a simple to use extension for three.js that activates all the rotating, dragging and zooming controls we need for both mouse and touch, there isn't a clear CDN for this that I can find -->
+    <script src='https://threejs.org/examples/js/controls/OrbitControls.js'></script>
+
+      <!-- Loads model-viewer for modern browsers -->
+    <script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.js"></script>
+    <!-- Loads model-viewer for older browsers -->
+    <script nomodule src="https://unpkg.com/@google/model-viewer/dist/model-viewer-legacy.js"></script>
+
+    <script src="script.js"></script>
```

Comment:
Likewise, this needs to be updated to reference your new javascript filename.

`<script src="webapp_script.js"></script>`

Reply:
kk updated - next time just tell me I'm wrong, no need to sugarcoat it jajajajajajajaja